src/platform/backend/drm.c

```diff
@@ -487,6 +571,8 @@ update_outputs(struct chck_pool *outputs)
 bool
 wlc_drm(struct wlc_backend *backend)
 {
+   const char *use_egldevice = getenv("WLC_USE_EGLDEVICE");
+   drm.use_egldevice = !chck_cstr_is_empty(use_egldevice);
```

Comment:
Maybe consider https://github.com/Cloudef/wlc/blob/f4ff049cc30242eec9f33af7129d2e2a2865fb5a/src/platform/render/gles2.c#L910 this way instead rather than just checking env var has something set.